docs/how-to/deploy.md

```diff
@@ -0,0 +1,8 @@
+# Deploy on the Cloud
+
+```{toctree}
+:maxdepth: 1
+
+docker-compose
```

Comment:
Docker compose is not being deployed in the cloud actually.

Docker compose provides just better and smoother local packaging and running, but is technically not deploying in the cloud

Reply:
sure i will just call it Deploy